### DISCUSSION TITLE
Aiflow on OCP : no plugins loaded

### DISCUSSION THREAD

**Discussion by moimeme27** [apache/airflow#32022] [Q&A]
hello, 
i'have mount airflow on OCP with shared NFS FS for Dags and plugins.
Dags is OK, but plugins is Not OK. 
on terminal on OCP : airflow plugins => no loaded plugins.
but always on terminal OCP : 
cd /opt/airflow/plugins => ls => i can see my files .py

How can i activate plugins on Airflow IHM ? 

thanks for your help

**Comment by potiuk** [comment]
Hard to say. your description does not give enough information. I think you need to debug as this is something on your side, and it's impossible to help you without logging it on your side

Some useful docs:

* https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/plugins.html#plugins
* https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/modules_management.html#modules-management

Generally following the docs and debugging (similarly to any other Python app) should give you the answers.

Good luck.
